elements/lisk-p2p/test/functional/actions/send.ts

```diff
@@ -84,12 +84,15 @@ describe('P2P.send', () => {
 
 	it('should receive a message in the correct format', async () => {
 		const firstP2PNode = p2pNodeList[0];
+
+		const connectedPeers = firstP2PNode.getConnectedPeers().length;
+
 		firstP2PNode.send({ event: 'bar', data: 'test' });
 
-		await wait(100);
+		await wait(300);
```

Comment:
I think setting arbitrary delays for testing creates brittle tests. Because what if the action takes a bit longer. If not why wasting time waiting :D I think, there should be a better way to make sure the message was transferred.

Reply:
I found that, some unreliable (we have no lifetime check after we emiting it into the network) operation like the send better to add some wait() to be sure that every peer receive and handle it. This could be ~5-100ms. 300ms is a safe value.